src/YoutubeAtom.tsx

```diff
@@ -246,95 +266,50 @@ export const YoutubeAtom = ({
     const [hasUserLaunchedPlay, setHasUserLaunchedPlay] = useState<boolean>(
         false,
     );
+    const [player, setPlayer] = useState<YoutubePlayerType | undefined>();
```

Comment:
I don't think we need to put the player in state. This component won't rerender so it can just be a scoped variable I guess?